README.md

```diff
@@ -30,3 +30,5 @@ If you're curious about the tools, stacks and technologies we use, get ready for
 - apply directly via [our hiring page](http://carrot.is/hiring)
 - [file an issue](https://github.com/carrot/we-are-hiring/issues) with a question you might have
 - fork this repository and PR in your resume/application (or just something we might find interesting)
+
+Last Updated: 04-10-2015 08:27 AM
```

Comment:
we won't merge the `origin/bump` branch into master and it'll just `--amend` the commit each time. just the minimum to keep it at the top of the list